### ISSUE TITLE
Eval CorefBERT in DocRED dataset

### ISSUE THREAD

**Issue by kabbi159** [thunlp/CorefBERT#3]
I tried to eval in DocRED dataset with usage in README.md, but I failed to eval.

Here is my execution in single GPU

preprocess:
```
python gen_data.py --model_type bert --model_name_or_path bert-base-cased --data_dir docred_data --output_dir prepro_data --max_seq_length 512
```

train:
```
CUDA_VISIBLE_DEVICES=0 python train.py --model_type bert --model_name_or_path ../CorefBERT_base  --train_prefix train --test_prefix dev --evaluate_during_training_epoch 5 --prepro_data_dir prepro_data --max_seq_length 512 --batch_size 16 --learning_rate 4e-5 --num_train_epochs 200 --save_name DocRED_CorefBERT_base
```

train result:
```
ALL  : Theta 0.5431 | F1 0.5745 | AUC 0.5434
Ignore ma_f1 0.5521 | input_theta 0.5431 test_result F1 0.5516 | AUC 0.5111
| epoch 199 | time: 165.89s
```

test:
```
CUDA_VISIBLE_DEVICES=0 python test.py --model_type bert --model_name_or_path ../CorefBERT_base --test_prefix test --prepro_data_dir prepro_data --max_seq_length 512 --batch_size 16 --save_name DocRED_CorefBERT_base --input_threshold 0.5431
```

But I get this result in test
```
Reading testing data...
test
Finish reading
total_recall 0
predicted as zero 380143
total ins num 391624
top1_acc 0
ma_f1 0.0000 | input_theta 0.5431 test_result F1 0.0000 | AUC 0.0000
finish output
Ignore ma_f1 0.0000 | input_theta 0.5431 test_result F1 0.0000 | AUC 0.0000
```

I compare model's state_dict before load model checkpoint, and there are differences between them. (So, I think there is no problem in load model's state_dict)
I tried to find where prediction in test dataset is failed, but I failed.

I did not modify any code, only difference is in **CUDA_VISIBLE_DEVICES=0 and batch_size 16**
(Ubuntu 18.04, RTX 3090, CUDA 11.0, Pytorch 1.7.0+cu110)

I need some help to eval this model.
Thanks

**Comment by YeDeming** [comment]
Hi, the test set is hidden. The script will generate a result.json and you should submit it to the colab sever for test score.

**Comment by kabbi159** [comment]
I forgot about that... Thanks for fast response about this issue!
